pages/resources/insights.mdx

```diff
@@ -0,0 +1,185 @@
+# Insights
+
+Insights are a data monitoring and analytics tool that Discord provides to help community managers understand their server's performance and how their members are interacting with it.
+
+### Insights Object
+
+Interval values are used to describe the time period over which the insights data is shown.
+
+#### Interval Values
```

Comment:
```suggestion
###### Aggregation Interval
```
h6 for structures/enums/flags 99% of the time, also keep it singular
(the real name for this was found in the dev portal btw if you're curious)